3party/freetype/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ set(FT_DISABLE_HARFBUZZ ON)
 
 add_subdirectory(freetype)
 
+# Fix warning with ONE_PIXEL macro clash.
+target_compile_options(freetype PRIVATE -Wno-macro-redefined)
```

Comment:
As an alternative turn-off unity for freetype?
Strange to have 2 different macro constants values.

Reply:
I didn't find a way to do it without modification of freetype's submodule's CMakeLists.txt upstream.

I've checked freetype's code, and it's safe to silence this warning. There is only one clash, and it uses correct values.